### ISSUE TITLE
Is MapBox Token needed if we dont make any Mapbox queries?

### ISSUE THREAD

**Issue by omarojo** [mapbox/mapbox-arkit-ios#34]
All I want is to position 3d objects in GPS coordinates. I dont need mapbox maps or directions or anything like that. Do I still need a mapbox token and internet connection for this purposes ?

**Comment by francisjervis** [comment]
No, a Mapbox API token is not required. You will likely need some connectivity for location services to work well, however.

**Comment by omarojo** [comment]
Thanks.. I implemented the mapbox positioning classes independently.. and apparently they work.  :)
